api/default_networks.go

```diff
@@ -75,8 +75,8 @@ func sepolia(stageName string) params.Network {
 		DefaultRPCURL:          fmt.Sprintf("https://%s.api.status.im/nodefleet/ethereum/sepolia/", stageName),
 		DefaultFallbackURL:     fmt.Sprintf("https://%s.api.status.im/infura/ethereum/sepolia/", stageName),
 		DefaultFallbackURL2:    fmt.Sprintf("https://%s.api.status.im/grove/ethereum/sepolia/", stageName),
-		RPCURL:                 "https://sepolia-archival.rpc.grove.city/v1/",
-		FallbackURL:            "https://sepolia.infura.io/v3/",
+		RPCURL:                 "https://sepolia.infura.io/v3/",
```

Comment:
if proxy can be in non-`DefaultUrl`, maybe rename all of them to RpcUrl1, RpcUrl2, RpcUrl3, RpcUrl4, RpcUrl5 ?